src/__tests__/downshift.lifecycle.js

```diff
@@ -99,6 +99,22 @@ test('props update of selectedItem will update the inputValue state', () => {
   )
 })
 
+test('props update of highlightedIndex will update highlighted index', () => {
```

Comment:
I think this test passes without your changes. I think we should change this test to make sure that the scroll position changes properly. Might take a little more work..